Description: Allow building with older sphinx versions, which still seems to
 work without problems. This partly reverts upstream 78380b9720a3c.
Forwarded: not-needed

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,7 +56,7 @@ include(pgr/Configure)
 # minimum versions
 #---------------------------------------------
 set(DOXYGEN_MINIMUM_VERSION "1.7")
-set(SPHINX_MINIMUM_VERSION "4.0")
+set(SPHINX_MINIMUM_VERSION "1.8")
 set(POSTGRESQL_MINIMUM_VERSION "9.2.0")
 set(BOOST_MINIMUM_VERSION "1.56.0")
 set(POSTGIS_MINIMUM_VERSION "2.0.0")
